### PR TITLE
Update model config generator tool

### PR DESCRIPTION
**Pull request by yewentao256**
## Motivation and Context
- 更新DIOPI model config生成工具，适配了国春对算子日志输出的更新，修复了一些已知问题
- 优化DIOPI model config生成工具，加强了日志输出与错误检查，提升了鲁棒性和运行效率
- 添加了模型真实使用场景的算子测例。

## Checklist
**Before PR**:

- [x] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Review by yangbofun — APPROVED** [review]

File: diopi_test/python/conftest.py [hunk 1]
```diff
@@ -7,19 +7,14 @@
...
 def init_dev():
-    global init_counter
-    init_counter += 1
-    print(f'[Device Init Times] {init_counter} .........')
     diopi_rt_init()
 
 
 def pytest_addoption(parser):
-    parser.addoption('--impl_folder', type=str, default='', help='folder to find device configs')
+    parser.addoption('--excel_path', type=str, default='report.xlsx', help='folder to find device configs')
```
Comment (jingguo-st) [inline diopi_test/python/conftest.py:17]: 换成--case-report-path？--case-result-path？

Comment (jingguo-st) [reply diopi_test/python/conftest.py:17]: 这里换的是--impl_folder吗？

Comment (LeungChiNan) [reply diopi_test/python/conftest.py:17]: impl_folder在这里是没用的，之前忘记删了，这个是最后生成的测试结果的excel文件路径，感觉可以改成--test_result_path

Comment (LeungChiNan) [reply diopi_test/python/conftest.py:17]: impl_folder在这里是没用的，之前忘记删了，这个是最后生成的测试结果的excel文件路径，感觉可以改成--test_result_path

**Review by jingguo-st — APPROVED** [review]

**Pull request closed without merging**